index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html> 
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/style.css">
+    <title> Index Document </title>
+</head>
+<body>
+      <nav>
+          <ul class="nav-list">
+            <li><a href="http://127.0.0.1:5500/Home.html"> Home </a></li>
+            <li><a href="http://127.0.0.1:5500/about.html"> About </a></li>
+            <li><a href="http://127.0.0.1:5500/portfolio.html"> Portfolio </a></li>
+          </ul>
+        </nav>
+        <header> 
+          <h1> Alaere Nekekpemi </h1>
+    </header>
+<main>
+    <header>
```

Comment:
`section` would be a better match here since this `p` text feels more like content than a header.